bolt-modules/boltlib/tasks/apply_catalog.rb

```diff
@@ -0,0 +1,57 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+# frozen_string_literal: true
+
```

Comment:
Do we want to put this somewhere else so it doesn't show up as a public task?

Reply:
That seems probably good, but I wasn't sure how best to accomplish it. If the intent is for boltlib to be "internal", we could redact it from the `bolt task show` output.

Or we could have this task live somewhere else and only be added to the modulepath temporarily while we're executing an `apply` block. That seems a bit more heavyweight.

Reply:
Redacting from `bolt task show` makes sense to me.

Reply:
It seems like we could bypass the run_task puppet function and searching the modulepath and just create a "task object" to hand to the executor directly?

Reply:
I'll give that a try.